go/perf/suite/suite.go

```diff
@@ -132,6 +132,14 @@ type perfSuiteT interface {
 	Suite() *PerfSuite
 }
 
+type environment struct {
```

Comment:
environment... Should we upper case that?

On Sun, Aug 21, 2016, 19:05 Aaron Boodman <notifications@github.com> wrote:

> In go/perf/suite/suite.go
> <https://github.com/attic-labs/noms/pull/2392#discussion_r75610466>:
>
> > @@ -132,6 +132,14 @@ type perfSuiteT interface {
> >  	Suite() *PerfSuite
> >  }
> >
> > +type environment struct {
>
> Will the Noms type name come out as Environment or environment?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/attic-labs/noms/pull/2392/files/ff54f886eb78b0759879abc9117155533b8bc140#r75610466>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACzFWj4dEH-aEOuR1vr452DLnnGQlqgks5qiQPygaJpZM4Jo5P0>
> .
>